src/lib/tableland-calls.ts

```diff
@@ -128,13 +130,19 @@ async function validateWriteQuery(
   return camelCaseKeys(json.result);
 }
 
-async function read(this: Connection, query: string): Promise<ReadQueryResult> {
+async function read(
+  this: Connection,
+  query: string,
+  options: ReadOptions = {}
+): Promise<ReadQueryResult> {
+  if (!options.output) options.output = "table";
```

Comment:
Sorry I didn't get to reviewing this earlier. The default behavior is defined on the validator and is `objects`. I don't think we want to override that here.

Reply:
Oh I now see this is to force the old behavior for existing users, sounds good, nice call!